src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,173 @@
+---
+title: Get Started
+layout: reference
+---
+
+# Request
+Concur Request automates the spend request and approval process for both travel and everyday expenses, giving you the data you need to accurately track and better control spending. By increasing visibility into planned expenses and up-to-date budget data, you can make strategic spending decisions before any spending actually occurs. The Request resource provides many abilities, including requests creation and transition into the approval workflow.
+
+## Get Started
+
+- [Overview](#overview)
+- [Version](#version)
+  - [Overview version V4.0](#overviewV4)
+- [Regional Availability](#regional)
+  - [With Swagger](#swagger)
+  - [With Curl](#curl)
+- [Explore the API](#explore-api)
+  - [Prerequisites](#prerequisites)
+  - [Acquire an Access Token](#token)
+  - [Explore the API in JavaScript](#javascript)
+    - [Retrieve a User Access Token](#retrieve-token)
+    - [Post a Request](#post-request)
+
+
+### <a name="overview"></a>Overview
+
+The Request V4 API has 4 resources:
```

Comment:
The Request V4 API contains four different resources:

Reply:
DONE with "The Request V4 API exposes four different resources:"